src/basicio.cpp

```diff
@@ -1093,11 +1093,16 @@ namespace Exiv2 {
     void MemIo::Impl::reserve(long wcount)
     {
         long need = wcount + idx_;
+        long    blockSize =     32*1024;   // 32768           `
+        long maxBlockSize = 4*1024*1024;
 
         if (!isMalloced_) {
-            // Minimum size for 1st block is 32kB
-            long size  = EXV_MAX(32768 * (1 + need / 32768), size_);
+            // Minimum size for 1st block
+            long size  = EXV_MAX(blockSize * (1 + need / blockSize), size_);
             byte* data = (byte*)std::malloc(size);
+            if (  data == NULL ) {
+                throw(60);
```

Comment:
You are throwing an integer here, I guess you want to do the following: `throw Error(60);`

Reply:
Of course.  I'm so confused and baffled by Git that I've overlooked the obvious blunder.

It has taken hours to submit this change.  I can't understand why git/github is so complicated.

Reply:
I think to make this change you just have to switch to the branch where you made the previous change, correct the two lines, do a `git commit` and `git push`. You could also amend the changes with `git commit --amend` but that will require a `git push -f` (which can be scary, as you can overwrite history with that).